Editing and previewing Markdown (#410)
obsidianmd/obsidian-docs: en/Editing and formatting/Editing and previewing Markdown.md

---
alias: How to/Read and edit modes
---

Obsidian lets you customize how to edit and preview notes with Markdown syntax using _editor views_ and _editor modes_.

Markdown is a markup language that uses special syntax to format text that's mainly useful when editing your notes. Less so when reading them. Instead, Obsidian can convert them into a format that's more suitable for reading.

Editor views and modes lets you customize how Markdown syntax appears when you're editing and reading notes.

## Editor views

> You can switch between the _editing view_ a _reading view_, depending on whether you intend to make changes to the note.

Author: Looks like this one managed to sneak through my linter. Good catch!

---
alias: How to/Read and edit modes
---

Obsidian lets you customize how to edit and preview notes with Markdown syntax using _editor views_ and _editor modes_.

> Markdown is a markup language that uses special syntax to format text that's mainly useful when editing your notes. Less so when reading them. Instead, Obsidian can convert them into a format that's more suitable for reading.

Reviewer: > Instead, Obsidian can convert them into a format that's more suitable for reading.

I'd say "Obsidian can show them in a way that's more suitable for reading".

I get what you mean. Obsidian _is_ rendering Markdown into HTML on the fly, but I feel like the word "convert" is an overkill here.

Author: ```suggestion
Markdown is a markup language that uses special syntax to format text that's mainly useful when editing your notes. Less so when reading them. Instead, Obsidian can show them in a way that's more suitable for reading.
```

Much better!
